### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by RaptaG**
I did my best!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-
-Alternatively, join my discord server: https://discord.gg/quvzkaV
+![](https://cdn-raw.modrinth.com//data/YBz7DOs8/images/49eab46d1e74e8f781404a8f773a1a09dedda5aa.png)
+_Screenshot taken with the [Awesome Skies](https://github.com/heymanMC/Awesome-Skies) resource pack, a pack made to run **only** under FSB_ (Credit: [heyman](https://github.com/heymanMC))
 
 ## Purpose
 
-This mod allows specification of custom skyboxes, with any textures that you may want.
-You can even specify textures already in Minecraft's resources!
+This mod allows the specification of Custom Skyboxes, with any textures that you may want, even with the ones which are already in Minecraft's resources!
```
Comment (AMereBagatelle) [inline README.md]: Probably just remove the bit after the second comma, it doesn't contribute much and isn't helpful information (I don't even know why I thought it was)

File: README.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+
+## Usage
 
-## Use
+### FabricSkyboxes' format
```
Comment (AMereBagatelle) [inline README.md]: It is grammatically correct but doesn't end up looking that nice in my opinion, possibly `FSB Skybox Format` or similar?  (get rid of the possessive I think)

File: README.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
-Documentation on the json format for skyboxes can be found [here](docs).
+Documentation on this format can be found [here](docs).
 
-##### Disclaimer:  Does not support Optifine skybox resource packs.  This is not planned.
+### OptiFine's format
 
-There are multiple projects that support converting Optifine skybox packs to FabricSkyboxes packs.  Please read their descriptions on how to use them.
-- MCPatcherPatcher (no longer maintained): https://github.com/LambdAurora/MCPatcherPatcher
-- MCPatcherPatcher++: https://github.com/supsm/MCPPPP
+It is **out of the scope** of this mod to support the OptiFine's format. [MCPP++](https://github.com/supsm/MCPPPP) can be used to get this packs to work and you can read its [instructions](https://github.com/supsm/MCPPPP#use) on how to do that.
```
Comment (AMereBagatelle) [inline README.md]: `the Optifine's` -> `Optifine's`
`this packs` -> `Optifine skyboxes`

**Review by AMereBagatelle — CHANGES_REQUESTED** [review]
Looks pretty good in general, just a few grammatical fixes and a few notes.  Thanks for submitting the PR!

**Comment by RaptaG** [comment]
> Looks pretty good in general, just a few grammatical fixes and a few notes.  Thanks for submitting the PR!

(Forgot to mention) I applied all the changes you requested!

**Review by AMereBagatelle — APPROVED** [review]

**Pull request merged**